canonical-web-and-design/ubuntu.com
# Create openstack chart

## Done

- Creates pie-chart representing number of commit by company to openstack

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/openstack/what-is-openstack
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare against [copy-doc](https://docs.google.com/document/d/15xrgIq_KUwHaCyFoI7-yS_JmBbsoyXJ4GpRMFAa7_is/edit?ts=60a3e5c8#)


## Issue / Card

Fixes https://github.com/canonical-web-and-design/ubuntu.com/issues/9782
